app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -714,7 +714,7 @@ class Result extends React.Component {
           submission_id={this.state.submission_id}
           assignment_id={this.state.assignment_id}
           grouping_id={this.state.grouping_id}
-          assignment_max_mark={this.state.assignment_max_mark}
+          assignment_max_mark={parseFloat(this.state.assignment_max_mark)}
```

Comment:
@WeissSa This does work but I don't think this is the root of the issue. Either the data from the server isn't being sent as a number (so we should fix it on the controller end) or it is, but then is being converted into a string when stored as state in this component.